lib/progressrus/store/redis.rb

```diff
@@ -59,7 +59,11 @@ def flush(scope, id = nil)
       private
 
       def key(scope)
-        "#{prefix}:#{scope.join(":")}"
+        if prefix.respond_to?(:call)
```

Comment:
Should we make `scope` either a `Proc` or a `String` instead? From the caller's perspective that's a more transparent API, isn't it?